validation/emailValidation.js

```diff
@@ -0,0 +1,20 @@
+let string = 'bellshade';
+let mail = 'bellshade@gmail.com';
+let pattern = /^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$/;
```

Comment:
Buat ngefix codacynya sih yang single quote diganti jadi double quote, yang `let` sengaja diganti `const` karena nilainya ga bakalan diubah ubah

```suggestion
const string = "bellshade";
const mail = "bellshade@gmail.com";
const pattern = /^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$/;
```

Reply:
good idea. wait

Reply:
done 👌 @reacto11mecha

Reply:
Oke, tinggal tunggu docs team